Assignment 6/Ques3.2.js

```diff
@@ -0,0 +1,40 @@
+// 3.2 Create a class Student that inherits from the Person class and has a property studies. Override the greeting() method to include information about what the student is studying.
+
+class Person{
+    name;
+    age;
+    gender;
+    interest;
+    constructor(name,age,gender,interest)
+    {
+       this.name=name;
+       this.age=age;
+       this.gender=gender;
+       this.interest=interest;
+    }
+    greeting()
+    {
+       return "Hi my name is : "+this.name+"\nAge is "+this.age+"\nGender: "+this.gender+"."+"\nInterest: "+this.gender;
```

Comment:
In the last line, `"+"\nInterest: "+this.gender`, you are using `gender` to show interest; it should be `"+"\nInterest: "+this.interest`.